scripts/stats/package.json

```diff
@@ -15,6 +15,7 @@
   "dependencies": {
     "axios": "^1.3.5",
     "date-fns": "^2.29.3",
-    "npm-stats-api": "^1.2.3"
+    "npm-stats-api": "^1.2.3",
+    "prettier": "^2.8.8"
```

Comment:
Is `prettier` being used here?

Reply:
Only by my editor which prompts me to install it if missing. I've removed it from this project.